pkg/virtual/workspaces/registry/org.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package registry
+
+import (
+	"time"
+
+	rbacinformers "k8s.io/client-go/informers/rbac/v1"
+	rbacv1client "k8s.io/client-go/kubernetes/typed/rbac/v1"
+	rbacv1listers "k8s.io/client-go/listers/rbac/v1"
+
+	tenancyclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/typed/tenancy/v1alpha1"
+	workspaceinformer "github.com/kcp-dev/kcp/pkg/client/informers/externalversions/tenancy/v1alpha1"
+	frameworkrbac "github.com/kcp-dev/kcp/pkg/virtual/framework/rbac"
+	workspaceauth "github.com/kcp-dev/kcp/pkg/virtual/workspaces/auth"
+)
+
+func NewOrg(orgRBACClient rbacv1client.RbacV1Interface, orgClusteWorkspaceClient tenancyclient.ClusterWorkspaceInterface, orgRBACInformers rbacinformers.Interface, orgCRBInformer rbacinformers.ClusterRoleBindingInformer, orgClusterWorkspaceInformer workspaceinformer.ClusterWorkspaceInformer) Org {
+	orgSubjectLocator := frameworkrbac.NewSubjectLocator(orgRBACInformers)
+	orgReviewerProvider := workspaceauth.NewAuthorizerReviewerProvider(orgSubjectLocator)
+
+	orgWorkspaceAuthorizationCache := workspaceauth.NewAuthorizationCache(
+		orgClusterWorkspaceInformer.Lister(),
+		orgClusterWorkspaceInformer.Informer(),
+		orgReviewerProvider.ForVerb("get"),
+		orgRBACInformers,
+	)
+
+	var stopCh chan struct{}
+
+	orgWorkspaceAuthorizationCache.Run(1*time.Second, stopCh)
+
+	return Org{
+		rbacClient:                orgRBACClient,
+		crbInformer:               orgCRBInformer,
+		crbLister:                 orgCRBInformer.Lister(),
+		workspaceReviewerProvider: orgReviewerProvider,
+		clusterWorkspaceClient:    orgClusteWorkspaceClient,
+		clusterWorkspaceLister:    orgWorkspaceAuthorizationCache,
+		stopCh:                    stopCh,
+		authCache:                 orgWorkspaceAuthorizationCache,
+	}
+}
+
+type Org struct {
+	// rbacClient can modify RBAC rules
+	rbacClient rbacv1client.RbacV1Interface
+	// crbInformer allows listing or seaching for RBAC cluster role bindings
+	crbInformer rbacinformers.ClusterRoleBindingInformer
+	// crbLister allows listing RBAC cluster role bindings
+	crbLister rbacv1listers.ClusterRoleBindingLister
+	// clusterWorkspaceClient can modify KCP workspaces
+	clusterWorkspaceClient tenancyclient.ClusterWorkspaceInterface
```

Comment:
all the client and informer comments above can gog. They are obvious.

Reply:
Fixed in commit c48becc80ef541475a0e032a26cfa6a9024c5ac8